server/src/models/BookFavorites.js

```diff
@@ -0,0 +1,43 @@
+const Model = require("./Model");
+
+class BookFavorites extends Model {
+  static get tableName() {
+    return "bookFavorites";
+  }
+
+  static get jsonSchema() {
+    return {
+      type: "object",
+      required: ["bookId", "bookListId"],
+      properties: {
+        title: { type: "string", minLength: 1 },
+        Author: { type: "string", minLength: 1 },
+      },
+    };
+  }
+
+  static get relationMappings() {
+    const { Book, BookList } = require("./index.js");
+
+    return {
+      books: {
+        relation: Model.HasManyRelation,
+        modelClass: Book,
+        join: {
+          from: "books.id",
+          to: "bookFavorites.bookId",
+        },
+      },
+
+      bookLists: {
+        relation: Model.HasManyRelation,
+        modelClass: BookList,
+        join: {
+          from: "bookLists.id",
+          to: "bookFavorites.bookListId",
+        },
+      },
+    };
```

Comment:
okay, I can see that you are guessing. These relations should be BelongstoOne, and the values in `from` and `to` need to be swapped. 

I know youve been reviewing, but journey back to the associations article and give it another pass